Number_types/include/CGAL/Filtered_rational.h

```diff
@@ -414,17 +414,17 @@ public :
     }
 };
 
-namespace NTC_INTERN{
+namespace FR_INTERN{
```

Comment:
Would you mind respecting the naming guideliens while you are at renaming the namespace?